packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -33,6 +33,39 @@ class JavascriptMessage {
 /// Callback type for handling messages sent from Javascript running in a web view.
 typedef void JavascriptMessageHandler(JavascriptMessage message);
 
+/// Information about a navigation action that is about to be executed.
+class NavigationRequest {
+  NavigationRequest({this.url, this.isMainFrame});
+
+  /// The URL a navigation is requested to.
```

Comment:
this doc doesn't pass our style guide ("url" and "navigation" and "request" are in the member and class name, so that leaves just "the a is ed to" which isn't meaningful).

Reply:
Gave it another shot.